awscli/customizations/s3/subcommands.py

```diff
@@ -649,7 +665,7 @@ class CpCommand(S3TransferCommand):
             "or <S3Uri> <S3Uri>"
     ARG_TABLE = [{'name': 'paths', 'nargs': 2, 'positional_arg': True,
                   'synopsis': USAGE}] + TRANSFER_ARGS + \
-                [METADATA_DIRECTIVE, EXPECTED_SIZE, RECURSIVE]
+                [METADATA, METADATA_DIRECTIVE, EXPECTED_SIZE, RECURSIVE]
```

Comment:
We should add ``METADATA`` to ``mv`` as well. @jamesls what is your thoughts on adding to ``sync`` as well? I think I would be in favor of adding it if someone had custom metadata that applied to all objects being synced.